improver/cli/neighbour_finding.py

```diff
@@ -186,16 +186,16 @@ def process(orography: cli.inputcube,
 
         all_methods = iris.cube.CubeList([])
         for method in methods:
-            all_methods.append(NeighbourSelection(**method).process(*fargs))
+            all_methods.append(NeighbourSelection(**method)(*fargs))
 
         squeezed_cubes = iris.cube.CubeList([])
         for index, cube in enumerate(all_methods):
             cube.coord('neighbour_selection_method').points = np.int32(index)
             squeezed_cubes.append(iris.util.squeeze(cube))
 
-        result = merge_cubes(squeezed_cubes)
+        result = MergeCubes()(squeezed_cubes)
```

Comment:
Have you got rid of all of the `merge_cubes` calls?  Because if so that wrapper function can be removed.  (Don't spend extra time checking if you didn't intend this, though.)

Reply:
You're way ahead of me!  Thanks for this tidy-up.